add v0.5.0 changelog (#452)
streamnative/function-mesh: CHANGELOG.md

# Function Mesh v0.5.0 Release Notes

## v0.5.0 What's New

> - [Test] Add tests for packages download ([#429](https://github.com/streamnative/function-mesh/pull/429))

Reviewer: ```suggestion
- [Test] Add tests for package download ([#429](https://github.com/streamnative/function-mesh/pull/429))
```

> - [Test] refine test cases of the Sink/Source/Crypto function ([#435](https://github.com/streamnative/function-mesh/pull/435))

Reviewer: ```suggestion
- [Test] Refine the test cases of the sinks, sources, and Crypto functions ([#435](https://github.com/streamnative/function-mesh/pull/435))
```

> - [Test] Use v1.8.2 for cert-manager tests ([#433](https://github.com/streamnative/function-mesh/pull/433))

Reviewer: ```suggestion
- [Test] Use cert-manager v1.8.2 for tests ([#433](https://github.com/streamnative/function-mesh/pull/433))
```

> - [Helm Charts] Add handle logic for FunctionMesh to clean up orphaned subcomponents ([#431](https://github.com/streamnative/function-mesh/pull/431))

Reviewer: ```suggestion
- [Helm Charts] Add the handle logic for Function Mesh to clean up orphaned subcomponents ([#431](https://github.com/streamnative/function-mesh/pull/431))
```

> - [Controller] Add the ability to set logging levels ([#445](https://github.com/streamnative/function-mesh/pull/445))

Reviewer: ```suggestion
- [Controller] Support setting the logging levels ([#445](https://github.com/streamnative/function-mesh/pull/445))
```

> - [Controller] Update CSV for openshift ([#417](https://github.com/streamnative/function-mesh/pull/417))

Reviewer: ```suggestion
- [Controller] Update the CSV for OpenShift ([#417](https://github.com/streamnative/function-mesh/pull/417))
```

> - [Controller] Add sky-walking workflow for self-signed tls pulsar cluster ([#439](https://github.com/streamnative/function-mesh/pull/439))

Reviewer: ```suggestion
- [Controller] Add the sky-walking workflow for the Pulsar cluster enabled with the self-signed TLS certificate ([#439](https://github.com/streamnative/function-mesh/pull/439))
```

- [Controller] Add a limit to the resource name ([#437](https://github.com/streamnative/function-mesh/pull/437/files))

> - [Function Mesh Worker service] Support subscriptionPosition for function&sink ([#322](https://github.com/streamnative/function-mesh-worker-service/pull/322))

Reviewer: ```suggestion
- [Function Mesh Worker service] Support the `subscriptionPosition` for functions and sinks ([#322](https://github.com/streamnative/function-mesh-worker-service/pull/322))
```

> - [Function Mesh Worker service] Support the `subscriptionPosition` for functions and sinks ([#322](https://github.com/streamnative/function-mesh-worker-service/pull/322))

Reviewer 2: please remove the mesh worker service changes in function-mesh's changelog.
```suggestion
```
